tools/windows/DatadogAgentInstaller/CustomActions/Native/RegistryServices.cs

```diff
@@ -15,5 +15,16 @@ public IRegistryKey CreateRegistryKey(Registries registry, string path)
 
             return new RegistryKey(key.CreateSubKey(path));
         }
+
+        public IRegistryKey OpenRegistryKey(Registries registry, string path)
+        {
+            var key = registry switch
+            {
+                Registries.LocalMachine => Registry.LocalMachine,
+                _ => null
+            };
```

Comment:
Curious expression, had to check the doc. Why cannot we use Registry.LocalMachine. What is is there. Would not be ternary expression sufficient?